File: Ejercicio6_Condicionales.cpp
/*6. Escriba un programa que lea de la entrada estándar un caracter e indique en la salida estándar si el caractér es una vocal minúscula, es una vocal mayúscula o no es una vocal.*/

#include<iostream>
using namespace std;

int main(){
    char letra;

    cout<<"Digite un caracter: "; cin>>letra;

    switch(letra){
        case 'a':
        case 'e':
        case 'i':
        case 'o':
        case 'u': cout<<"\nEs una vocal minuscula"<<endl; break;
        case 'A':
        case 'E':
        case 'I':
        case 'O':
        case 'U': cout<<"\nEs una vocal mayuscula"<<endl; break;
        default: cout<<"\nNo es una vocal"<<endl; break;
    }

    return 0;
}

File: Comparativa Python/Ejercicio6-Condicionales.py
# 6. Escriba un programa que lea de la entrada estándar un caracter e indique
# en la salida estándar si el caractér es una vocal minúscula, es una vocal
# mayúscula o no es una vocal.

caracter = input('Digite un caracter: ')
if caracter == 'a' or caracter == 'e' or caracter == 'i' or caracter == 'o' or caracter == 'u':
    print('Es una vocal minúscula.')
elif caracter == 'A' or caracter == 'E' or caracter == 'I' or caracter == 'O' or caracter == 'U':
    print('Es una vocal mayúscula.')
else:
    print('El caracter digitado NO es una vocal')
